rune/libenclave/internal/runtime/pal/pal_linux.go

```diff
@@ -32,7 +36,12 @@ func (pal *enclaveRuntimePal) Attest() (err error) {
 }
 
 func (pal *enclaveRuntimePal) Exec(cmd []string, envp []string, stdio [3]*os.File) (int32, error) {
-	api := &enclaveRuntimePalApiV1{}
+	if pal.version <= 1 {
```

Comment:
Change this to pal.version == 1. In order to handle pal.version == 0, add a check in getPalApiVersion().